components/channel_view/index.js

```diff
@@ -29,11 +31,16 @@ function mapStateToProps(state) {
     const config = getConfig(state);
     const enableTutorial = config.EnableTutorial === 'true';
     const tutorialStep = getInt(state, Preferences.TUTORIAL_STEP, getCurrentUserId(state), TutorialSteps.FINISHED);
+    const channel = getCurrentChannel(state);
+    const isReadOnly = channel && channel.name === Constants.DEFAULT_CHANNEL &&
```

Comment:
Could you make this into a selector function? I think you'll need this in the mobile repo as well, so it could live in the redux repo. It could be something like `isCurrentChannelReadOnly`